build.sbt

```diff
@@ -178,9 +179,30 @@ lazy val oidcHbp = project
                                 scalaTest % Test)
   )
 
+lazy val elastic = project
```

Comment:
do we really need a new module here? seems we can build things in the "service" and avoid the multi-module hassle

Reply:
For testing right now is better. It wasn't working for now with the service test dependencies